### PR TITLE
update mdbook function links to 0.0.3

### PR DESCRIPTION
**Pull request by Connoropolous**

### CODE REVIEW

File: doc/holochain_101/src/zome/api_functions.md [hunk 1]
```diff
@@ -137,39 +137,47 @@ Entry lookup is done in the following order:
 Caller can request additional metadata on the entry such as type or sources
 (hashes of the agents that committed the entry).
 
-[View it in the Rust HDK](https://developer.holochain.org/api/latest/hdk/api/fn.get_entry.html)
+- [View get_entry in the Rust HDK](https://developer.holochain.org/api/0.0.3/hdk/api/fn.get_entry.html)
+- [View get_entry_initial in the Rust HDK](https://developer.holochain.org/api/0.0.3/hdk/api/fn.get_entry_initial.html)
+- [View get_entry_initial in the Rust HDK](https://developer.holochain.org/api/0.0.3/hdk/api/fn.get_entry_initial.html)
```
Comment (pdaoust) [inline doc/holochain_101/src/zome/api_functions.md]: looks like this line is in there twice.

Comment (Connoropolous) [reply doc/holochain_101/src/zome/api_functions.md]: Ah, thanks. That should say 'get_entry_history'

File: doc/holochain_101/src/zome/api_functions.md [hunk 2]
```diff
@@ -137,39 +137,47 @@ Entry lookup is done in the following order:
...
 Consumes three values, two of which are the addresses of entries, and one of which is a string that defines a relationship between them, called a `tag`. Later, lists of entries can be looked up by using `get_links`. Entries can only be looked up in the direction from the `base`, which is the first argument, to the `target`, which is the second.
 
-[View it in the Rust HDK](https://developer.holochain.org/api/latest/hdk/api/fn.link_entries.html)
+[View it in the Rust HDK](https://developer.holochain.org/api/0.0.3/hdk/api/fn.link_entries.html)
 
 ### Query
 
 Canonical name: `query`
 
 Returns a list of addresses of entries from your local source chain, that match a given entry type name, or a vector of names. You can optionally limit the number of results, and you can use "glob" patterns such as "prefix/*" to specify the entry type names desired.
```
Comment (pdaoust) [inline doc/holochain_101/src/zome/api_functions.md]: looks like there's an unescaped asterisk here which is making the rest of the line italic. Not related to the links updates, sorry, but thought I'd mention it while it was in front of me :)

**Comment by Connoropolous** [comment]
"Is it easy to change these links en masse with each release?" Yes

**Comment by pdaoust** [comment]
> "Is it easy to change these links en masse with each release?" Yes

Cool; figured it was just one `grep | sed -i` away :+1:

**Pull request merged**